scripts/fuzz_opt.py

```diff
@@ -1673,6 +1673,13 @@ def ensure(self):
         tar.extractall(path=self.clusterfuzz_dir)
         tar.close()
 
+    def can_run_on_wasm(self, wasm):
+        # Do not run ClusterFuzz in the first seconds of fuzzing: the first time
+        # it runs is very slow (to build the bundle), which is annoying when you
+        # are just starting the fuzzer and looking for any obvious problems.
+        seconds = 30
+        return time.time() - start_time > seconds
```

Comment:
Alternatively, we could wait 100 fuzzer iterations if that information is already available.

Also, will this mess with being able to reproduce an error with a particular seed?

Reply:
The number of iterations is present somewhere but it would be slightly more work to get it.

This does prevent reproducing an error with a seed, but ClusterFuzz is already not fully reproducible, as it really tries to recreate the production environment, including using the same random generation as them:

https://github.com/WebAssembly/binaryen/blob/613ea882e07fb9ed777f46a26678213bce212927/scripts/fuzz_opt.py#L1600-L1602

Reply:
makes sense, thanks 👍